### PR TITLE
added flag and pep8

### PR DESCRIPTION
**Pull request by Nolski**

### CODE REVIEW

File: fart/models/__init__.py [hunk 1]
```diff
@@ -4,9 +4,11 @@
 
 Base = declarative_base()
 
+
 def initialize_sql():
     Session = sessionmaker()
-    engine = create_engine('sqlite:///:memory:', echo=True)
+    engine = create_engine('sqlite:///mypoop.db', echo=True,
```
Comment (dxa4481) [inline fart/models/__init__.py]: yo broski, not on master, master is clean

**Comment by Nolski** [comment]
@dxa4481 I cleaned it up. My most sincere apologies.

**Comment by dxa4481** [comment]
Learn to wipe your ass @Nolski

**Comment by Nolski** [comment]
lesson learned. Do I get a ship it

**Comment by dxa4481** [comment]
and then don't wipe after

**Pull request merged**